manage.py

```diff
@@ -59,6 +59,7 @@ def init_hq_python_path():
         'ptop_preindex',
         'sync_prepare_couchdb_multi',
         'sync_couch_views',
+        'celery',
```

Comment:
ok - i think it might be better to only do this if -P gevent are also in the list of args